fastlane/Fastfile

```diff
@@ -102,6 +102,27 @@ end
 
 desc "Installs all Certs and Profiles necessary for development and ad-hoc"
 lane :match_me do
+  run_match_me(readonly: true)
+end
+
+desc "Register new device, regenerates profiles, updates them remotely and locally"
+lane :register_new_device do
+  device_name = prompt(text: "Enter the device name: ")
+  device_udid = prompt(text: "Enter the device UDID: ")
+
+  register_device(
+    name: device_name,
+    udid: device_udid
+  )
+
+  run_match_me(readonly: false)
+end
```

Comment:
I like this lane! However the name should reflect that profiles will be recreated